app/controllers/settings_controller.rb

```diff
@@ -1,24 +1,13 @@
-class SettingsController < ApplicationController
+class SettingsController < Api::V2::BaseController
```

Comment:
should we move the controller to API then? :thinking: 
I'm not sure it's a good thing to inherit API, here...

Reply:
Never mind, get it from ^^, consider it irrelevant for this PR :)